.appveyor.yml

```diff
@@ -0,0 +1,44 @@
+platform:
+  - x64
+
+branches:
+  only:
+  - master
+
+environment:
+
+  matrix:
+    - TARGET_ARCH: x86
+      PYVER: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      PYVER: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x86
+      PYVER: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
+
+    - TARGET_ARCH: x64
+      PYVER: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
+
+    - TARGET_ARCH: x86
+      PYVER: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36
+
+    - TARGET_ARCH: x64
+      PYVER: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
+
+install:
+  - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+  - cmd: conda.exe install -yq conda=4.3
```

Comment:
The Python 3.5-based Miniconda on AppVeyor has `conda` 4.2, which is too old to install the latest `constructor`. So we need to ensure that we install `conda` 4.3 for that case. The Python 2.7-based and Python 3.6-based Minicondas on AppVeyor already have `conda` 4.3. Also we need to ensure we don't go to `conda` 4.4 as that will break `constructor`.